### ISSUE TITLE
Uppercase.py

### ISSUE THREAD

**Issue by jinymusim** [ufal/npfl114#96]
args.logdir not in base template, making people lose data by not saving it.
It's masked by the 
os.environ.setdefault("TF_CPP_MIN_LOG_LEVEL", "2") # Report only TF errors by default
which i think should NOT by used.

**Comment by foxik** [comment]
I am sorry, I do not understand.
- the `os.environ.setdefault("TF_CPP_MIN_LOG_LEVEL", "2")` limits TensorFlow logs to errors (ignoring warnings and infos); but if incluences only TensorFlow
- `args.logdir` is initialized in https://github.com/ufal/npfl114/blob/9b93f15f6c7d48ac59950465a1731bdeec421829/labs/03/uppercase.py#L30-L34
- it is then used when saving the predictions in https://github.com/ufal/npfl114/blob/9b93f15f6c7d48ac59950465a1731bdeec421829/labs/03/uppercase.py#L65-L66

The idea is that multiple runs are saved in different directories (with their names containing date, time and hyperparameters of that run).

What exactly is the problem?

**Comment by jinymusim** [comment]
As writen file with treated text didn't save, after removing args.logdir (which now i chcecked is indeed in my code) and join on path, i was able to save the text. Before, the file just disappeared

**Comment by foxik** [comment]
If no exception was raised, then the file was most likely written. But it was written to `args.logdir`, so it should be in the `logs/...` subdirectory, not in the current directory.

**Comment by jinymusim** [comment]
Yes! Thank you!
